app/home/route.ts

```diff
@@ -20,6 +26,8 @@ export default class Home extends Route {
 
   @action
   didTransition() {
-      this.analytics.trackPage();
+      const shouldShowVersionB = this.features.isEnabled(camelize(ABTesting.homePageVersionB));
```

Comment:
Turns out `camelize()` is not necessary here because `ember-feature-flags` does this internally.
```suggestion
      const shouldShowVersionB = this.features.isEnabled(ABTesting.homePageVersionB);
```